test/functional/template_test.rb

```diff
@@ -0,0 +1,82 @@
+require File.join(File.dirname(__FILE__), 'test_helper')
+
+describe 'template' do
+  describe 'clone' do
+    before do
+      @cmd = %w(template clone)
+    end
+
+    it 'should print error on missing --id' do
+      params = ['--new-name=zzz']
+
+      expected_result = CommandExpectation.new
+      expected_result.expected_err =
+        ['Could not clone the config template:',
+         "  Missing arguments for 'id'",
+         ''].join("\n")
+      expected_result.expected_exit_code = HammerCLI::EX_USAGE
+
+      api_expects_no_call
+
+      result = run_cmd(@cmd + params)
+
+      assert_cmd(expected_result, result)
+    end
+
+    it 'should print error on missing --new-name' do
+      params = ['--id=1']
+
+      expected_result = CommandExpectation.new
+      expected_result.expected_err =
+        ['Could not clone the config template:',
+         "  Missing arguments for 'config_template[name]'",
```

Comment:
This default message isn't very helpful. Could you please change it to a message that points users to `--new-name` ? This parameter is always required so I thing the easiest way to achieve this is:
```ruby
validate_options do
  option(:option_new_name).required
end
```